src/net.cpp

```diff
@@ -2053,7 +2053,7 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
             if (anchor && !m_anchors.empty()) {
                 const CAddress addr = m_anchors.back();
                 m_anchors.pop_back();
-                if (m_anchors.size() == 0) delete_anchors_file = true;
+                if (m_anchors.empty()) delete_anchors_file = true;
```

Comment:
nit: in order to create minimal diffs (and save reviewers time), I'd suggest to stick to either `.size() == 0` or `.empty()` from the very first commit that introduces it and not change it after.